server/pkg/model/model_test.go

```diff
@@ -374,7 +381,7 @@ func TestModel_SetPublic(t *testing.T) {
 		args args
 	}{
 		{
-			name: "test",
+			name: "set public true",
 			args: args{
 				public: true,
```

Comment:
public: false

We want to test if the field is correctly changed:

True -> false or false -> true